client/package.json

```diff
@@ -13,52 +13,52 @@
-    "@angular/forms": "~11.2.4",
-    "@angular/material": "~11.2.3",
-    "@angular/platform-browser": "~11.2.4",
-    "@angular/platform-browser-dynamic": "~11.2.4",
-    "@angular/router": "~11.2.4",
+    "@angular/animations": "~12.2.15",
+    "@angular/cdk": "~12.2.13",
+    "@angular/common": "~12.2.15",
+    "@angular/compiler": "~12.2.15",
+    "@angular/core": "~12.2.15",
+    "@angular/flex-layout": "~12.0.0-beta.35",
+    "@angular/forms": "~12.2.15",
+    "@angular/material": "~12.2.13",
+    "@angular/platform-browser": "~12.2.15",
+    "@angular/platform-browser-dynamic": "~12.2.15",
+    "@angular/router": "~12.2.15",
     "rxjs": "~6.6.6",
     "tslib": "^2.0.0",
-    "zone.js": "~0.10.3"
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1102.3",
-    "@angular-eslint/builder": "1.2.0",
-    "@angular-eslint/eslint-plugin": "1.2.0",
-    "@angular-eslint/eslint-plugin-template": "1.2.0",
-    "@angular-eslint/schematics": "1.2.0",
-    "@angular-eslint/template-parser": "1.2.0",
-    "@angular/cli": "~11.2.3",
-    "@angular/compiler-cli": "~11.2.4",
-    "@angular/language-service": "~11.2.4",
+    "@angular-devkit/architect": "^0.1301.3",
+    "@angular-devkit/build-angular": "~12.2.15",
+    "@angular-eslint/builder": "~12.7.0",
+    "@angular-eslint/eslint-plugin": "~12.7.0",
+    "@angular-eslint/eslint-plugin-template": "~12.7.0",
+    "@angular-eslint/schematics": "~12.7.0",
+    "@angular-eslint/template-parser": "~12.7.0",
+    "@angular/cli": "~12.2.15",
+    "@angular/compiler-cli": "~12.2.15",
+    "@angular/language-service": "~12.2.15",
     "@nrwl/cypress": "^12.6.3",
     "@nrwl/workspace": "^11.4.0",
-    "@types/jasmine": "^3.6.4",
-    "@types/jasminewd2": "~2.0.3",
-    "@types/node": "^13.13.45",
-    "@typescript-eslint/eslint-plugin": "4.3.0",
-    "@typescript-eslint/parser": "4.3.0",
+    "@types/jasmine": "^3.8.2",
+    "@types/node": "^16.4.10",
+    "@typescript-eslint/eslint-plugin": "^5.9.1",
+    "@typescript-eslint/parser": "^5.9.1",
     "cypress": "^6.6.0",
     "cypress-mongo-seeder": "^1.0.8",
-    "eslint": "^7.21.0",
-    "eslint-plugin-import": "2.22.1",
-    "eslint-plugin-jsdoc": "30.7.6",
-    "eslint-plugin-prefer-arrow": "1.2.2",
-    "jasmine-core": "~3.6.0",
-    "jasmine-spec-reporter": "~5.0.0",
-    "karma": "~6.1.1",
+    "eslint": "^7.32.0",
+    "eslint-plugin-import": "2.25.4",
+    "eslint-plugin-jsdoc": "36.0.6",
+    "eslint-plugin-prefer-arrow": "1.2.3",
+    "jasmine-core": "~3.10.1",
+    "jasmine-spec-reporter": "~7.0.0",
+    "karma": "~6.3.11",
     "karma-chrome-launcher": "~3.1.0",
-    "karma-coverage-istanbul-reporter": "~3.0.2",
-    "karma-jasmine": "~4.0.0",
-    "karma-jasmine-html-reporter": "^1.5.4",
-    "ts-node": "~8.3.0",
-    "typescript": "~4.0.7"
+    "karma-coverage": "^2.1.0",
+    "karma-jasmine": "~4.0.1",
+    "karma-jasmine-html-reporter": "^1.7.0",
+    "ts-node": "^9.1.1",
```

Comment:
I would upgrade to ts-node 10.x.x, but that seemed to break our e2e tests.

(At least, I think it was ts-node that was causing the issue. It may have been an intermittent failure.)

Reply:
Do we want to create an issue to remind us to look into this later? Or is Dependabot already fussing about it?

Reply:
We certainly could. (I haven't looked into it enough to know what's up. 🤷🏻‍♂️)

I think dependabot will keep fussing about it, though.